rca/project_styleguide/templates/patterns/molecules/key-details/key-details--event.html

```diff
@@ -0,0 +1,119 @@
+            </li>
+        </ul>
+    </div>
+
+    {% if page.who_can_attend %}
+    <div class="key-details__section key-details__section--who">
+        <h3 class="body body--two key-details__sub-heading">Who {% if page.past %}could{% else %}can{% endif %} attend</h3>
+        <ul class="key-details__list">
+            <li class="key-details__list-item">
+                {{ page.who_can_attend }}
+            </li>
+        </ul>
+    </div>
+    {% endif %}
+
+    {% if taxonomy_tags %}
+        <div class="key-details__section key-details__section--tags">
+            <h3 class="body body--two key-details__sub-heading">School, Centre or Area</h3>
+            <ul class="key-details__list">
+                {% for item in taxonomy_tags %}
+                    <li class="key-details__list-item">
+                        <a class="link link--tertiary" href="{{ item.href }}">{{ item.title }}</a>
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+    {% endif %}
+
+    {% if page.type %}
+    <div class="key-details__section key-details__section--location">
+        <h3 class="body body--two key-details__sub-heading">Type</h3>
+        <ul class="key-details__list">
+            <li class="key-details__list-item">
+                {{ page.type }}
+            </li>
+        </ul>
+    </div>
+    {% endif %}
+
+    {% if page.series.items %}
+        <div class="key-details__section key-details__section--tags">
+            <h3 class="body body--two key-details__sub-heading">Series</h3>
+            <ul class="key-details__list">
+                {% for item in page.series.items %}
+                    <li class="key-details__list-item">
+                        <a class="link link--tertiary" href="{{ item.link }}">{{ item.title }}</a>
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+    {% endif %}
+
+    <div class="key-details__section key-details__section--read-time">
+        <h3 class="body body--two key-details__sub-heading">Read time</h3>
+        <ul class="key-details__list">
+            <li class="key-details__list-item">
+                {{ page.read_time }}
+            </li>
+        </ul>
+    </div>
+
+    {% if page.add_to_calendar or page.schedule %}
+        <div class="key-details__section key-details__section--press-pack">
+            <ul class="key-details__list">
+                {% if page.schedule %}
+                <li class="key-details__list-item">
+                    <a href="#" class="link link--tertiary">
+                        <span class="link__label">View schedule</span>
+                    </a>
+                </li>
+                {% endif %}
+                {% if page.add_to_calendar %}
+                <li class="key-details__list-item">
+                    <a href="#" class="link link--tertiary link--link link--external" target="_blank">
```

Comment:
does this need `rel="noopener noreferrer"` adding if it's going to an external link? appreciate there's no href yet : )